modules/apps/wysiwyg/wysiwyg-web/src/main/java/com/liferay/wysiwyg/web/internal/upgrade/WysiwygWebUpgrade.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.wysiwyg.web.internal.upgrade;
+
+import com.liferay.dynamic.data.mapping.service.DDMStructureLocalService;
+import com.liferay.portal.kernel.module.framework.ModuleServiceLifecycle;
+import com.liferay.portal.kernel.upgrade.DummyUpgradeStep;
+import com.liferay.portal.kernel.upgrade.UpgradeException;
+import com.liferay.portal.upgrade.registry.UpgradeStepRegistrator;
+import com.liferay.portal.upgrade.release.BaseUpgradeWebModuleRelease;
+import com.liferay.wysiwyg.web.internal.upgrade.v1_0_0.UpgradeContentPortlet;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Peter Fellwock
+ */
+@Component(immediate = true, service = UpgradeStepRegistrator.class)
+public class WysiwygWebUpgrade implements UpgradeStepRegistrator {
```

Comment:
When will this upgrade run?

I tend to think that we should make it so that it's run automatically when the app is deployed. What do you think?